notes/interview.vbs

```diff
@@ -8752,7 +8953,10 @@ EMReadScreen MX_region, 10, 22, 48
 MX_region = trim(MX_region)
 If MX_region = "INQUIRY DB" Then
 	continue_in_inquiry = MsgBox("You have started this script run in INQUIRY." & vbNewLine & vbNewLine & "The script cannot complete a CASE:NOTE when run in inquiry. The functionality is limited when run in inquiry. " & vbNewLine & vbNewLine & "Would you like to continue in INQUIRY?", vbQuestion + vbYesNo, "Continue in INQUIRY")
-	If continue_in_inquiry = vbNo Then Call script_end_procedure("~PT Interview Script cancelled as it was run in inquiry.")
+	If continue_in_inquiry = vbNo Then
+		STATS_manualtime = STATS_manualtime + (timer - start_time)
+		Call script_end_procedure("~PT Interview Script cancelled as it was run in inquiry.")
+	End If
 End If
 If MX_region = "TRAINING" Then developer_mode = True
 
```

Comment:
Line 9013 Spelling Error: Canceled should be "cancelled"

 Text 75, 135, 255, 10, "This means that IF the script errors, fails, is canceled, the network goes down."

Reply:
Line 9018 & 1716 Spelling Error: writen should be "written"

Reply:
9198 Spelling Error: "confrim with the resident should be "confirm"

Reply:
Spelling errors fixed.